src/cpp/fastdds/subscriber/DataReaderImpl.hpp

```diff
@@ -234,6 +234,31 @@ class DataReaderImpl
     ReturnCode_t get_subscription_matched_status(
             SubscriptionMatchedStatus& status);
 
+    /**
+     * @brief Retrieves in a publication associated with the DataWriter
+     *
+     * @param[out] publication_data publication data struct
+     * @param publication_handle InstanceHandle_t of the publication
+     * @return RETCODE_BAD_PARAMETER if the DataReader is not matched with
+     * the given publication handle, RETCODE_OK otherwise.
```

Comment:
```suggestion
     * @param publication_handle @ref InstanceHandle_t of the publication
     * @return @ref RETCODE_BAD_PARAMETER if the DataReader is not matched with
     * the given publication handle, @ref RETCODE_OK otherwise.
```